service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerSubmittedTest.java

```diff
@@ -238,6 +247,77 @@ void shouldNotifyManagedLAWhenCaseCreatedOnBehalfOfLA() {
+            .legalRepresentation(YES.getValue())
+            .solicitor(RespondentSolicitor.builder()
+                .firstName(SOLICITOR_FIRST_NAME)
+                .lastName(SOLICITOR_LAST_NAME)
+                .email(SOLICITOR_EMAIL)
+                .unregisteredOrganisation(UnregisteredOrganisation.builder()
+                    .name("Unregistered Org Name")
+                    .build())
+                .build()).build();
+
+        CaseDetails caseDetails = populatedCaseDetails(Map.of("id", CASE_ID));
+        caseDetails.getData().put("respondents1", wrapElements(respondent));
+        caseDetails.getData().put("caseLocalAuthorityName", LOCAL_AUTHORITY_1_NAME);
+
+        postSubmittedEvent(buildCallbackRequest(caseDetails, OPEN));
+
+        String expectedSalutation = String.format("Dear %s %s", SOLICITOR_FIRST_NAME, SOLICITOR_LAST_NAME);
+
+        Map<String, Object> expectedUnregisteredSolicitorParameters = mapper.convertValue(
+            RespondentSolicitorTemplate.builder()
+                .salutation(expectedSalutation)
+                .localAuthority(LOCAL_AUTHORITY_1_NAME)
+                .build(),
+            new TypeReference<>() {});
+
+        checkUntil(() ->
+            verify(notificationClient).sendEmail(
+                UNREGISTERED_RESPONDENT_SOLICICTOR,
+                SOLICITOR_EMAIL,
+                expectedUnregisteredSolicitorParameters,
+                NOTIFICATION_REFERENCE
+            ));
+    }
+
+    @Test
+    void shouldNotNotifyUnregisteredSolicitorWhenUnregisteredOrganisationDetailsNotProvided() {
+        Respondent respondent = Respondent.builder()
+            .legalRepresentation(YES.getValue())
+            .solicitor(RespondentSolicitor.builder()
+                .firstName(SOLICITOR_FIRST_NAME)
+                .lastName(SOLICITOR_LAST_NAME)
+                .email(SOLICITOR_EMAIL)
+                .build()).build();
+
+        CaseDetails caseDetails = populatedCaseDetails(Map.of("id", CASE_ID));
+        caseDetails.getData().put("respondents1", wrapElements(respondent));
+        caseDetails.getData().put("caseLocalAuthorityName", LOCAL_AUTHORITY_1_NAME);
+
+        postSubmittedEvent(buildCallbackRequest(caseDetails, OPEN));
+
+        String expectedSalutation = String.format("Dear %s %s", SOLICITOR_FIRST_NAME, SOLICITOR_LAST_NAME);
+
+        Map<String, Object> expectedUnregisteredSolicitorParameters = mapper.convertValue(
+            RespondentSolicitorTemplate.builder()
+                .salutation(expectedSalutation)
+                .localAuthority(LOCAL_AUTHORITY_1_NAME)
+                .build(),
+            new TypeReference<>() {});
+
+        checkUntil(() ->
+            verify(notificationClient, never()).sendEmail(
+                UNREGISTERED_RESPONDENT_SOLICICTOR,
+                SOLICITOR_EMAIL,
+                expectedUnregisteredSolicitorParameters,
+                NOTIFICATION_REFERENCE
+            ));
+    }
```

Comment:
I think can just be 
`verifyNoInteractions(notificationClient);
`

Reply:
Not too sure about this one, a few emails are sent once the case is submitted which would cause the test to fail